applications/metis_application/custom_processes/set_mpi_communicator_process.h

```diff
@@ -115,6 +115,11 @@ class SetMPICommunicatorProcess : public Process
 
     void Execute() override
     {
+        KRATOS_WARNING("SetMPICommunicatorProcess")
+            << "Calling deprecated process SetMPICommunicatorProcess.\n"
+            << "Please use ModelPartCommunicatorUtilities::SetMPICommunicator(ModelPart) instead."
+            << std::endl;
+
```

Comment:
The big point is removing it from the MetisApplication, where it should have never been on the first place (my mistake). I transformed it to an Utility because it is a Process that only implements Execute, but I can leave it as a process if desired.